src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -25,14 +25,14 @@ namespace OpenLoco::GameCommands
 {
     namespace Flags
     {
-        constexpr uint8_t apply = 1 << 0;         // 0x01
-        constexpr uint8_t flag_1 = 1 << 1;        // 0x02
-        constexpr uint8_t flag_2 = 1 << 2;        // 0x04
-        constexpr uint8_t noErrorWindow = 1 << 3; // 0x08 do not show an error window even on failure (use this with ghosts)
-        constexpr uint8_t aiAllocated = 1 << 4;   // 0x10 ai can place down invisible ghosts with this that blocks players
-        constexpr uint8_t noPayment = 1 << 5;     // 0x20 calculates cost but does not deduct it
-        constexpr uint8_t ghost = 1 << 6;         // 0x40
-        constexpr uint8_t flag_7 = 1 << 7;        // 0x80
+        constexpr uint8_t apply = 1 << 0;              // 0x01
+        constexpr uint8_t skipGhostPlacement = 1 << 1; // 0x02
+        constexpr uint8_t skipFundingCheck = 1 << 2;   // 0x04
```

Comment:
i'd say it was more a allow going negative balance but tbh its not really used i wonder if we could jsut remove it.

Reply:
Oh, I really misunderstood then. Perhaps `Flags::allowNegativeCashFlow` would work as a new name?

If we're going to remove this, I think that would be better left to a different PR.